File: docs/examples/python/verify_token.py
import os
import requests

CORE_AUTH_BASE_URL = os.environ.get("CORE_AUTH_BASE_URL", "http://localhost:3000")


def verify_token(token: str) -> dict:
    res = requests.post(
        f"{CORE_AUTH_BASE_URL}/verify",
        json={"token": token},
        timeout=5,
    )
    res.raise_for_status()
    return res.json()

File: docs/examples/node/verify-token.ts
const CORE_AUTH_BASE_URL = process.env.CORE_AUTH_BASE_URL ?? 'http://localhost:3000';

export async function verifyToken(token: string) {
  const res = await fetch(`${CORE_AUTH_BASE_URL}/verify`, {
    method: 'POST',
    headers: { 'Content-Type': 'application/json' },
    body: JSON.stringify({ token }),
  });

  if (!res.ok) {
    throw new Error(`verify failed: ${res.status}`);
  }

  return res.json();
}
